package.json

```diff
@@ -69,7 +69,7 @@
     "rimraf": "3.0.2",
     "rollup": "^2.60.2",
     "rollup-plugin-terser": "^7.0.2",
-    "rollup-plugin-uglify": "^6.0.4",
+    "rollup-plugin-uglify": "^5.0.2",
```

Comment:
Any reason we're downgrading this and husky (above)?

Reply:
npm audit

Reply:
I suppose brevity is the soul of wit